src/main/java/server/Router.java

```diff
@@ -0,0 +1,19 @@
+package main.java.server;
+
+import main.java.server.request.Request;
+
+import java.util.HashMap;
+
+public class Router {
+
+    public String route(Request request) {
+        HashMap<String, String> requestHeaders = request.getHeaders();
+        String path = requestHeaders.get("path");
+        if (path.equals("/simple_get")) {
```

Comment:
Just want to flag this here since we previewed it a bit in the IPM - this is logic that a server should not strictly need to know (i.e. what paths merit a non-404 response). Definitely don't mind leaving it here until we've got some more routes in place, since that could give you a better idea of how and where to extract them - but want to point to a concrete example of where we can start thinking about app vs server.